backend/src/v1/services/userService.js

```diff
@@ -6,31 +6,25 @@ const getAllUsers = async () => {
   return await User.getAllUsers();
 };
 
-const getOneUser = () => {
-  return;
-};
-
 const createNewUser = async (newUser) => {
+  // console.log('here', newUser);
   const createdUser = {
     id: uuid(),
     ...newUser,
   };
+  console.log('Services:', newUser);
+  const newCheckedUser = await User.checkNewUser(createdUser);
+  if (newCheckedUser) {
+    console.log('username already used');
+    return false;
```

Comment:
```suggestion
    return 'Username is taken';
```
Let's return a string here for the username taken error and just pass it along from the controller.